install/Knative-custom-install.md

```diff
@@ -66,13 +66,13 @@ service mesh. If you install any of the following options, you must install
 
 #### Istio installation options
 
-| Istio Install Filename     | Description                                                            |
-| -------------------------- | ---------------------------------------------------------------------- |
-| **[`istio-crds.yaml`][a]** | Creates CRDs before installing Istio.                                  |
-| **[`istio.yaml`][b]**      | Install Istio with service mesh enabled (automatic sidecar injection). |
-| [`istio-lean.yaml`][c]     | Install Istio and disable the service mesh by default.                 |
+**The files in _bold italics_ are the recommended standard install files suitable for most use cases.**
```

Comment:
I don't know, I really like how scannable the table is this way. I do wish it was easier in markdown to show a table's legend, but I think it's hard to miss that bold line on top of the table, and as you look at each group of components, the bold italic ones jump out at you. I don't love the idea of repeating "Tip: compatible with most use cases" for each file where that's true.

How would you feel about using some other symbol (maybe †, if we can get that to show up in markdown) to denote this meaning, instead of just the formatting?

Reply:
Right now this line renders like a double title and the table holds 4 conventions (1.code and link for install file, 2.code bold italic and link for "common use case", 3. link to more info, 4. bold table section for component). Honestly, I think the trouble here is that our adv install guide is trying to also act as an install reference document(and help non adv users too). 

Maybe the solution is the maintain our adv. user audience focus. I think the adv user audience is going to know whether or not they want an ELK stack and what that means. To a new user, I dont think "suitable for most use cases"  (-> common use case?) is actually helpful due to the lack of detail (i immediately ask my self 'why?', what are our common use cases and are they not covered in the host specific install guides).

(temp fix) If you really want to combine new users info here too, we could clean up the table by simply adding a different indicator (†) but I really dont like the extra bold (header looking) line and would like that perhaps, moved to the end, under the table.

(long term solution) Or we could have a separate document that is focused on new users and better discuss each "use case" and what files to install.

Reply:
I don't think it hurts to call out what the "standard" or "common" install files are, even for advanced users. It's good for them to know what baseline they're deviating from. I went with option 1.

I feel like "a separate document focused on new users" already exists; the on-any-K8s install kind of serves that purpose. Or one of the many platform-specific guides.